resources/source-types/cockroach.yaml

```
@@ -0,0 +1,311 @@
+apiVersion: bindplane.observiq.com/v1
+kind: SourceType
+metadata:
+  name: cockroachdb
+  displayName: CockroachDB
+  icon: /icons/sources/cockroachdb.svg
+  description: Collect metrics and logs from CockroachDB.
+spec:
+  version: 0.0.1
+  supportedPlatforms:
+    - macos
+    - linux
+    - windows
+  parameters:
+  # Metrics
+  - name: enable_metrics
+    label: Enable Metrics
+    description: Enable to send metrics.
+    type: bool
+    default: true
+
+  - name: hostname
+    label: Hostname
+    description: The hostname or IP address of the CockroachDB system.
+    type: string
+    default: ""
+    required: true
+    relevantIf:
+      - name: enable_metrics
+        operator: equals
+        value: true
+
+  - name: port
+    label: Port
+    description: The TCP port of the CockroachDB system.
+    type: int
+    default: 8080
+    relevantIf:
+      - name: enable_metrics
+        operator: equals
+        value: true
+
+  - name: username
+    label: Username
+    description: The username to use when connecting to CockroachDB.
+    type: string
+    required: false
+    relevantIf:
+      - name: enable_metrics
+        operator: equals
+        value: true
+
+  - name: password
+    label: Password
+    description: The password to use when connecting to CockroachDB.
+    type: string
+    required: false
+    relevantIf:
+      - name: enable_metrics
+        operator: equals
+        value: true
+    options:
+      password: true
+
+  - name: collection_interval
+    label: Collection Interval
+    description: How often (seconds) to scrape for metrics.
+    type: int
+    default: 60
+    relevantIf:
+      - name: enable_metrics
+        operator: equals
+        value: true
+    advancedConfig: true
+
+  # Logging
+  #
+  - name: enable_logs
+    label: Enable_Logs
+    description: Enable to collect logs.
+    type: bool
+    default: true
+  - name: enable_health_log
```

nit, lets be consistent with spacing between each param.
```suggestion
    default: true
  
  - name: enable_health_log
```